x-pack/elastic-agent/pkg/core/plugin/state/state.go

```diff
@@ -4,22 +4,56 @@
 
 package state
 
-import "github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/plugin/process"
+import (
+	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/plugin/process"
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+)
 
 // Status describes the current status of the application process.
 type Status int
 
 const (
 	// Stopped is status describing not running application.
-	Stopped Status = iota
-	// Running signals that application is currently running.
+	Stopped Status = 0
```

Comment:
this way all the values are 0s. need to use iota

Reply:
fixing